pkg/surveyext/editor.go

```diff
@@ -104,7 +104,9 @@ func (e *GhEditor) prompt(initialValue string, config *survey.PromptConfig) (int
 			break
 		}
 		if r == '\r' || r == '\n' {
-			if e.BlankAllowed {
```

Comment:
Let me know if you'd like a nested if statement or just keep the flat if statement.

Reply:
You are welcome to restructure the conditional logic to something that would be more cleaner in your view.

Reply:
This is resolved in commit 91d4ae117d9704f1dc675c10222b722eab18b662.